Yoast/wordpress-seo
# Set proper size and viewBox values in the exclamation triangle SVG icon.

## Summary

This PR can be summarized in the following changelog entry:

* [not user-facing] Set proper values for the width, height, and viewBox attributes of the exclamation triangle SVG icon

## Relevant technical choices:

- as noted in https://github.com/Yoast/wordpress-seo-premium/issues/2405#issuecomment-495600404, after the changes from https://github.com/Yoast/wordpress-seo/pull/12827 we have to avoid to use the same values for these attributes
- the imagemin optimization will strip out the `viewBox` attribute if it contains the same values from the width and height attributes, e.g.: `width="1792" height="1792" viewBox="0 0 1792 1792"`

In any case, it's very unlikely we will ever want an icon to have a size of 1792 pixels.

Note: this is not necessary to fix https://github.com/Yoast/wordpress-seo-premium/issues/2405 but it's a best practice we should adopt consistently.

## Test instructions
- open the file `images/exclamation-triangle.svg`
- check there are these values: `width="16" height="16" viewBox="0 0 1792 1792"`
- run `grunt build:images`
- check the values `width="16" height="16" viewBox="0 0 1792 1792"` are still there

## UI changes


See https://github.com/Yoast/wordpress-seo-premium/issues/2405
